source/includes/_endpoint-projects.md

```diff
@@ -269,14 +312,41 @@ PATCH /projects/abcd/members/ HTTP/1.1
 ```json
 {
   "element": "shoji:catalog",
-  "self": "http:\/\/local.crunch.io:8080\/api\/projects\/6c01\/members\/",
+  "self": "http://beta.crunch.io/api/projects/6c01/members/",
   "index": {
-    "http:\/\/local.crunch.io:8080\/api\/users\/001\/": {},
-    "http:\/\/local.crunch.io:8080\/api\/users\/002\/": null
+    "http://beta.crunch.io/api/users/001/": {},
+    "http://beta.crunch.io/api/users/002/": {
+      "permissions": {
+        "edit": true
+      }
+    },
+    "http://beta.crunch.io/api/users/003/": null,
+    "user@email.com": {},
+    "send_notification": true,
+    "url_base": "https://beta.crunch.io/password/change/${token}/"
   }
 }
 ```
 
+###### Sending notifications
+
+The users invited to a project can be both existing Crunch.io users or new
+users that don't have a user account associated with the email.
+
+If desired, the API can send automated email notifications to the involved users
+indicating that they now belong to the project.
+
+It is necessary to add the `send_notification` boolean key on the index PATCHed
+to command the API to send these emails. Else, no notification will be sent.
+
+When sending notifications, it is necessary for the client to include a 
+`url_base` key as well that includes a string template that should point to a
+client location where the password resetting should happen for brand new users.
+
+The server will replace the `${token}` part of the string with the generated
+token and will be included on the notification email as a link for the invited 
+user to configure their account in order to use the app.
+
```

Comment:
Please add a link back to the sharing/permissions docs since this is the same idea as what's there.